### ISSUE TITLE
Cannot achieve the baseline using huggingface gpt-2 in pytorch version

### ISSUE THREAD

**Issue by liuchengyuan123** [czyssrs/Logic2Text#5]
Hello!
I am trying to reproduce your work in pytorch. For the gpt-2 baseline, I use the model from huggingface. I download it here: https://huggingface.co/gpt2. I have trained the ptm for 100+ epochs, but the BLEU result achieves 24 at most. Then the metric goes lower.
I copied all other training configurations from your code, except for the pretrained model. Could you please advice the possible solutions?
Thanks for your answer!
